src/openrct2/platform/linux.c

```diff
@@ -56,6 +56,8 @@ void platform_get_exe_path(utf8 *outPath, size_t outSize)
         log_fatal("failed to get process path");
 
     }
+#elif defined(__OpenBSD__)
+    strlcpy(exePath, "/usr/local/bin/", sizeof(exePath));
```

Comment:
How can you be sure it will always be `/usr/local/bin`? What about the case when user compiles it somewhere in his workspace? Even the FreeBSD part just above interrogates this information from the system dynamically.